### DISCUSSION TITLE
Migration problem using latest hasura and hasura cli v2.1.1

### DISCUSSION THREAD

**Discussion by dodanex** [hasura/graphql-engine#7970] [General]
Hello,

I have a problem with migrations using the latest version of hasura and hasura cli using config v3.

I am trying to migrate from google cloudrun:
Hasura v2.1.1
PostgresSQL v13.4
https://hasura-clone-yyljssc5aq-ey.a.run.app/console =  works fine
https://hasura-clone-yyljssc5aq-ey.a.run.app/healthz = results: 404 error

To my local environment:
Hasura cli v2.1.1
Hasura v2.1.1
PostgresSQL v13.4
http://localhost:8080/console = works fine
http://localhost:8080/healthz = results: OK

I’m following the official documentation steps on my local environment:

1. install hasura-cli locally via npm inside a folder: $ npm install --save-dev @aaronhayes/hasura-cli@latest

2. $ npx hasura version (results v2.1.1)

3. $ npx hasura init myproject --endpoint https://hasura-clone-yyljssc5aq-ey.a.run.app --admin-secret mysecret

4. $ cd myproject

5. npx hasura migrate create "init" --from-server

I get this errors:

ERRO connecting to graphql-engine server failedINFO possible reasons:           
INFO 1) Provided root endpoint of graphql-engine server is wrong. Verify endpoint key in config.yaml or/and value of --endpoint flag
INFO 2) Endpoint should NOT be your GraphQL API, ie endpoint is NOT https://hasura-cloud-app.io/v1/graphql it should be: https://hasura-cloud-app.io
INFO 3) Server might be unhealthy and is not running/accepting API requests
INFO 4) Admin secret is not correct/set

INFO time="2021-12-22T02:23:08+02:00" level=fatal msg="request failed: url: https://hasura-clone-yyljssc5aq-ey.a.run.app/healthz status code: 404 status: 404 Not Found"    

So the problem seems to be this https://hasura-clone-yyljssc5aq-ey.a.run.app/healthz = I get 404 error ☹

Is this a problem with hasura installation on my google cloudrun or is this a problem with hasura cli because of postgres version v13.4 ?

This is urgent for me and I really appreciate any help I can get !

**Comment by rikinsk** [comment]
Hi, this is indeed due to the fact that the /healthz endpoint is returning a 404. The CLI checks the /healthz endpoint before running any commands on a server and that check must be failing. 
I am not quite sure why your /healthz endpoint is returning a 404 though. Could it be a deployment configuration issue?

**Reply by dodanex** [reply]
Thanks for your answer @rikinsk !
I have now tested with a fresh new hasura installation on google cloud run, just to see if I can acces the /healthz on this one. The problem is the same, I get 404 error. This is so strange and I don't know how to fix this. I'll keep searching for a solution.
